### PR TITLE
Fix set examples ref in behave tests (#587)

### PR DESCRIPTION
**Pull request by radtriste**
Backport from https://github.com/kiegroup/kogito-images/pull/587

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: scripts/common.py [hunk 1]
```diff
@@ -212,7 +212,7 @@ def update_examples_ref_in_behave_tests(examples_ref):
     """
     print("Set examples_ref {} in behave tests".format(examples_ref))
     # this pattern will look for any occurrences of using master or using x.x.x
-    pattern = re.compile(r'(using nightly-master)|(using nightly-\s*([\d.]+.x))|(using \s*([\d.]+))')
+    pattern = re.compile(r'(using nightly-master)|(using nightly-\s*([\d.]+.x))|(using \s*([\d.]+[.x]?))')
```
Comment (spolti) [inline scripts/common.py:215]: is it to match the 1.9.x, for example?

Comment (radtriste) [reply scripts/common.py:215]: yes it is :)

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
@spolti to you for approve & merge :)

**Review by spolti — APPROVED** [review]

**Pull request merged**